pymc/__init__.py

```diff
@@ -63,7 +63,8 @@ def __set_compiler_flags():
     logsumexp,
     probit,
 )
-from pymc.model import *
+from pymc.model.core import *
+from pymc.model.transform.conditioning import *
```

Comment:
We only want `do` and `observe` at the root level. There are other functions there that shouldn't be imported